config/defaults/analytics/WEB-INF/classes/log4j.properties

```diff
@@ -1,6 +1,6 @@
 log4j.rootLogger=@shared.default.log.level@, R
 
-log4j.logger.org.georchestra.analytics=DEBUG, console
+log4j.logger.org.georchestra.analytics=WARN, R
```

Comment:
Should be @shared.default.log.level@ as for https://github.com/georchestra/georchestra/blob/14.06/config/defaults/catalogapp/WEB-INF/classes/log4j.properties#L3